src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -176,6 +176,7 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
 
         #region SerializeCore/DeserializeCore
 
+        [Obsolete]
```

Comment:
I must be missing something here. Could you explain this change.

Reply:
@aparajit-pratap `SerializeCore` is obsoleted in the parent class but not here throwing a warning so I added the tag